### PR TITLE
Update to 2.0.8

### PR DESCRIPTION
**Pull request by chujimmy**
@oliverviljamaa  : update Angular components for comparisons. Contains both layout : `ComparisonTable` and `ComparisonBarTable`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Angular components for comparison ([React comparison-components](https://github.
 ## Components
 
 ### `<tw-comparison-table />`
+<img src="http://i.imgur.com/IEAYQfH.png" width="800" />
+
+### `<tw-comparison-bar-table />`
```
Comment (oliverviljamaa) [inline README.md]: Do you think we even need the bar table anywhere? I think we'd want to have one version of a comparison table everywhere, wouldn't we?

File: README.md [hunk 2]
```diff
@@ -18,10 +21,24 @@ Angular components for comparison ([React comparison-components](https://github.
...
+    target="$ctrl.target"
+    amount="$ctrl.amount"
+    lang="$ctrl.lang"
+    provider-country="$ctrl.providerCountry"
+    source-country="$ctrl.sourceCountry"
+  >
+  </tw-comparison-bar-table>
 ```
-where `$ctrl.source` and `$ctrl.target` are currency codes (`GBP`, `EUR`, f.e.) and `$ctrl.amount` is the amount (`1000`, f.e.).
+where `$ctrl.source` and `$ctrl.target` are currency codes (`GBP`, `EUR`, f.e.), `$ctrl.amount` is the amount (`1000`, f.e.), `$ctrl.lang` the language to be use din the component ('en', 'de' and 'es' at the moment), `$ctrl.providerCountry` is the country (ISO-3) where the provider is (banks or non traditional banks such as Western Union) and `$ctrl.sourceCountry` the origin country of the funds (in order to get comparison from EUR to GBP from Spain and including non traditional banks).
```
Comment (oliverviljamaa) [inline README.md]: `used in`

File: package.json [hunk 3]
```diff
@@ -41,11 +41,12 @@
     "less": "^2.7.1",
     "less-loader": "^2.2.3",
     "ngreact": "^0.3.0",
-    "postcss-loader": "^1.1.0",
+    "postcss-loader": "^0.13.0",
```
Comment (oliverviljamaa) [inline package.json]: Interesting, why the older version?

File: src/comparison-bar-table/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+import ComparisonBarTableComponent from './comparison-bar-table.component';
+import './comparison-bar-table.less';
+
+const ComparisonBarTable = angular
+  .module('tw.comparison-components.comparison-bar-table', [
+    ngReact,
+  ])
+  .component('twComparisonBarTable', ComparisonBarTableComponent)
+  .value('ComparisonBarTableReact', ComparisonBarTableReact)
```
Comment (oliverviljamaa) [inline src/comparison-bar-table/index.js]: Why is this added to the DI?

File: README.md (hunk 1 above)
Comment (chujimmy) [inline README.md]: Yep, wanted to keep both versions to measure separately the impact of the new version and the new design. But I can remove it since the new design works better.

File: package.json (hunk 3 above)
Comment (chujimmy) [inline package.json]: Compatibility issues with webpack. Going to make another try to not downgrade

File: README.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ Angular components for comparison ([React comparison-components](https://github.
 ## Components
 
 ### `<tw-comparison-table />`
-<img src="https://i.imgur.com/bNtSMu6.png" width="800" />
+<img src="http://i.imgur.com/IEAYQfH.png" width="800" />
```
Comment (oliverviljamaa) [inline README.md]: I like hosting on GitHub now, feels a bit safer, WDYT? You can just paste an image in any `textarea` and it'll give you an URL you can use.

File: package.json (hunk 3 above)
Comment (oliverviljamaa) [inline package.json]: What was the issue? Maybe we can keep the same version and just add `postcss: {}` to Webpack config, this has helped for some previous issues if I remember correctly.

Comment (chujimmy) [inline package.json]: Yep, that was it
